.github/workflows/build-test-publish.yml

```diff
@@ -0,0 +1,41 @@
+name: Build, Test and maybe Publish
+on: [push, pull_request]
+
+jobs:
+  
+  test:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Use Node v12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Build
+        run: npm install
```

Comment:
Does the build step require a `npm run build` here for typescript, or does Github actions know to run that as a convention?

Reply:
Yeah good spot! This is not yet setup for the `build` + `bundle` step needed too to create something which can be consumed through a `script` tag. Adding later on!